src/debugger/launch.ts

```diff
@@ -254,12 +252,16 @@ export function createDarwinTestConfiguration(
     };
 }
 
+function launchConfigKeysToUpdate(): string[] {
+    return ["program", "cwd", "preLaunchTask", `env.${swiftLibraryPathKey()}`];
+}
```

Comment:
Do we still need this function? At least it's expressive and clear enough to put inline.